resmio_conversion.js

```diff
@@ -0,0 +1,195 @@
+    var queryString = url ? url.split('?')[1] : window.location.search.slice(1);
+    // variable (object) which stores the parameters
+    var obj = {};
+    // if query string exists
+    if (queryString) {
+      // stuff after # is not part of query string, so get rid of it
+      queryString = queryString.split('#')[0];
+      // split our query string into its component parts
+      var arr = queryString.split('&');
+      for (var i=0; i<arr.length; i++) {
+        // separate the keys and the values
+        var a = arr[i].split('=');
+        // in case params look like: list[]=thing1&list[]=thing2
+        var paramNum = undefined;
+        var paramName = a[0].replace(/\[\d*\]/, function(v) {
+          paramNum = v.slice(1,-1);
+          return '';
+        });
+        // set parameter value (use 'true' if empty)
+        var paramValue = typeof(a[1])==='undefined' ? true : a[1];
+        // (optional) keep case consistent
+        paramName = paramName.toString().toLowerCase();
+        paramValue = paramValue.toString().toLowerCase();
+        // if parameter name already exists
+        if (obj[paramName]) {
+          // convert value to array (if still string)
+          if (typeof obj[paramName] === 'string') {
+            obj[paramName] = [obj[paramName]];
+          }
+          // if no array index number specified...
+          if (typeof paramNum === 'undefined') {
+            // put the value on the end of the array
+            obj[paramName].push(paramValue);
+          }
+          // if array index number specified...
+          else {
+            // put the value at that index number
+            obj[paramName][paramNum] = paramValue;
+          }
+        }
+        // if param name doesn't exist yet, set it
+        else {
+          obj[paramName] = paramValue;
+        }
+      }
+    }
+    return obj;
+  }
+
+  /*
+   * Creates the cookie
+   * -------------------
+   */
+  function createCookie(name,value,days) {
+    // check if cookie already exists -> disabled
+    /*if (document.cookie.indexOf(name < 0) {*/
+      if (days) {
+        var date = new Date();
+        date.setTime(date.getTime()+(days*24*60*60*1000));
+        var expires = "; expires="+date.toGMTString();
+      }
+      else {
+        var expires = "";
+      }
+      document.cookie = name+"="+value+expires+"; path=/";
+    /*}*/
+  }
+
+  /*
+   * Reads a specific cookie and returns its value
+   * ---------------------------------------------
+   */
+  function readCookie(name) {
+    var nameEQ = name + "=";
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+      var c = ca[i];
+      while (c.charAt(0)==' ') c = c.substring(1,c.length);
+      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+    }
+    return null;
+  }
+
+  /*
+   * Deletes a specific cookie
+   * -------------------------
+   */
+  function eraseCookie(name) {
+    createCookie(name,"",-1);
```

Comment:
No way ... Jose ;-) rewrote the function to:

document.cookie = name+"=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/";

--> hope that is now the correct way to do it (at least W3C suggests it that way)